### PR TITLE
Move update filesizes job

### PR DESCRIPTION
**Pull request by hanifn**
## Description

Move the scheduled attachment filesize metadata updater into `a8c-files` so that it can run on all sites regardless of wether Stream Wrappers is active or not. This will allow us to run the job before activating the Stream Wrapper for a site and avoid the initial performance hit.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [x] This change works and has been tested on a Go sandbox.

### CODE REVIEW

**Comment by mjangda** [comment]
This will need a refresh as there have been lots of changes in the deploy repo (once those are stable).

**Comment by mjangda** [comment]
@hanifn I've synced over the changes. Can you take a look to make sure everything seems okay?

File: a8c-files.php [hunk 1]
```diff
@@ -832,6 +870,148 @@ function image_resize( $ignore, $id, $size ) {
...
+			'interval' => 180,
+			'display' => __( 'Once every 3 minutes, unlike what the slug says. Originally used to be 5 mins.' ),
+		];
+
+		return $schedule;
+	}
+
+	public function schedule_update_job() {
+		// Only schedule meta update job on production
+		if ( 'production' !== VIP_GO_ENV ) {
```
Comment (hanifn) [inline a8c-files.php]: @batmoo with the new `VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE` constant check I added, would this `ENV` be needed anymore? I feel like it's a better to leave that sort of details to deployment configs

Comment (mjangda) [reply a8c-files.php]: Yep, I think we can safely drop this.

I guess to for the rollout, we'd do something like this?

```
if ( ! defined( 'VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE' ) ) {
   if ( 'production' !== VIP_GO_ENV ) {
        define( 'VIP_FILESYSTEM_SCHEDULE_FILESIZE_UPDATE', true );
   }
}
```

Comment (hanifn) [reply a8c-files.php]: Yeap. This way we can easily deactivate or change batch sizes for individual sites should we need to

Comment (mjangda) [reply a8c-files.php]: Awesome, should be good to go after that change.

**Comment by mjangda** [comment]
r136845-deploy

**Pull request merged**